linode/manager
# feat: [M3-6867] - Betas Landing

## Description 📝
Adds the basic layout for the betas landing page and utility functions for working with `Beta` and AccountBeta objects.

## Major Changes 🔄
**List highlighting major changes**
- Creates `features/betas/index.tsx`
- Adds utility functions for working with `Beta` and `AccountBeta` objects

## Preview 📷
![image](https://github.com/linode/manager/assets/115022759/7708ce28-ae97-409c-85e8-9b1bb13af427)

## How to test 🧪
run `yarn test BetasLanding betaUtils`
Ensure that when you visit `/betas` the betas landing page is rendered
